### PR TITLE
TG-1492: Reduce screen / full screen behavior

### PR DESCRIPTION
**Pull request by jehadmelad**
TG-1492

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,2 @@
 /.github/  @youwol/continuous-integration
+/.github/  @youwol/dev-front
```
Comment (youwol-jdecharne) [inline .github/CODEOWNERS:2]: Only team @youwol/continuous-integration should have ownership of `/.github/`
Give team @youwol/dev-front ownership to _other_ part of the code (see [Github Documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax))

**Review by youwol-jdecharne — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jehadmelad) [reply .github/CODEOWNERS:2]: i will revert it as before

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-/.github/  @youwol/continuous-integration
+/.github/  @youwol/continuous-integration
```
Comment (youwol-jdecharne) [inline .github/CODEOWNERS:1]: Missing EOL at end of file

Comment (youwol-jdecharne) [inline .github/CODEOWNERS:1]: Missing EOL there.
Maybe juste rebase the whole thing to `drop` the only commit modifying `CODEOWNERS`:
```shell
# assuming the checkout branch is `feature/TG-1492` and remote `origin` is the Github repo
git rebase -i origin/main
```

**Review by youwol-jdecharne — CHANGES_REQUESTED** [review]

**Review by youwol-guillaume — DISMISSED** [review]

**Review by youwol-jdecharne — APPROVED** [review]

**Pull request merged**